### DISCUSSION TITLE
Feature proposal: Dungeons wen

### DISCUSSION THREAD

**Discussion by DaanMich** [sunflower-land/sunflower-land#2607] [Ideas]
Hi everyone I’ve had this idea in my head for quite a while now so just getting it out of my system here. 

**Dungeons wen?**
Each bumpkin would be able to embark on a legendary quest to a dungeon. A bumpkin would need to be equipped with weapons, food and even a boat/raft. A journey takes a full day and your bumpkin would require to rest as well after this long trip (this is where multiple bumpkins also shine). 

**Mechanic?**
Via the boat a player could click on « dungeon » and be able to embark on this legendary quest. For this to work a player would need to have a minimum of items to start travelling. E.g minimum one sword, one raft and a ton of food to get through the journey. You can craft these items in the blacksmith for obvious reasons.

**Used boosts?**
Armour: a bumpkin can survive longer in this horrific dungeon. Less food is needed to survive (50%). 

Goblin sword/ sunflower hammer: squash enemies with these weapons. Crafting swords/weapons is 10% cheaper. 

Warrior badge: output of the dungeon gets boosted with 30%. 

**Rewards?**
This is where a ton of unique rewards could be added for potion use. E.g. slime, spiders, string, moss,… 
Some other rewards could include finding pets along the journey or an animal (wen cows?). 

**Conclusion?**
I feel like this could be an amazing feature that gives current nfts a use and we can expand on by a lot. I’m fine to have a basic/raw feature first to expand it later to have for example bumpkins running in an actual dungeon (using phaser or other). 

*Disclaimer:*
This is just a basic proposal / all values here are made up and aren’t to be taken serious.

**Comment by brian-man** [comment]
> Warrior badge: output of the dungeon gets boosted with 30%.

I like the idea of applying some buff for Warrior -- both badges and wearables.

I dislike the idea of "general boost to dungeon", as I think that's too strong.  Maybe an increase in survival (less food required or whatever) would be good.

**Reply by DaanMich** [reply]
Boosts are entirely made up so it’s more the general concept I was trying to pitch. 

However I did try going of off supply and general availability of the items in question to determine some kind of baseline boost. Also wanted to try a unique boost per item. 

Ofc other things such as finding an animal/pet along the way could have increased odds. Or maybe overall decreased journey costs for the warrior badge (but feels less unique). 

Just to add some data on supply: 
- goblin sword / human hammer: ~ 2.5k
- Warrior armour: ~500 of each item (you would need full set so let’s say /3 = 166)
- warrior badge: 90 

This is why the proposed values were what they were but still all made up ofc.

**Comment by NotKeen** [comment]
I like the general idea. There could also be an opportunity to let players collaborate on these dungeon raids. You could specialize for different classes / roles. Ofc this is yet another RNG mechanic, which some people might not like. Personally I think RNG adds some spice to games. And the possibility to find super rare items is exciting to many (including myself) I think :D

**Reply by DaanMich** [reply]
Really like that idea as well! Could include rankings singleplayer and multiplayer or could also include guild raids as Dcol said. Lots of opportunities to expand this feature!

**Comment by TheDcol** [comment]
Rewards for potion use or evenfor guild islands 👀
